src/app/report/edit-display/edit-display.component.ts

```diff
@@ -36,25 +39,32 @@ import { NgIf, NgFor } from '@angular/common';
     CustomEditorComponent,
     DisplayTableComponent,
     NgFor,
+    BooleanToStringPipe,
+    NgStyle,
+    ClipboardModule,
   ],
 })
 export class EditDisplayComponent {
   editingEnabled: boolean = false;
   editingChildNode: boolean = false;
   editingRootNode: boolean = false;
+  metadataTableVisible: boolean = false;
   rerunResult: string = '';
   report: any = {};
+  displayReport: boolean = false;
   @Input() id: string = '';
-  currentView: any = {
-    storageName: 'Test',
-  };
-  @Output() saveReportEvent = new EventEmitter<any>();
+  @Input() containerHeight!: number;
+  @Input() currentView: any = {};
+  @Input() newTab: boolean = true;
+  @Output() saveReportEvent: Subject<any> = new Subject<any>();
```

Comment:
subject containing any?

Reply:
Again typing everything is done in another PR, therefore I did not do anything about types